term-utils/scriptreplay.c

```diff
@@ -143,9 +152,12 @@ int
 main(int argc, char *argv[])
 {
 	static const struct timeval mindelay = { .tv_sec = 0, .tv_usec = 100 };
+	static const struct timeval inputDelay = { .tv_sec = 0, .tv_usec = 100000 };
+	struct timeval stepDelay = { 0, 0 };
```

Comment:
Please keep the variable names consistent. We rarely use C++-like names like "fooBar;" it seems better to use "inputdelat" or "input_delay."

Reply:
Fixed with https://github.com/util-linux/util-linux/pull/3015/commits/28d7c58404ec8b38021f90bd8930783f0279ebe0